sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -117,6 +115,23 @@ class EthService(
       }
     )
 
+  private def convertMempoolMapInspect(poolMap: TrieMap[SidechainTypes#SCP, MempoolMap#TxByNonceMap]) =
+    CollectionConverters.mapAsJavaMap(
+      poolMap.map { case (proposition, txByNonce) =>
+        new Address(proposition.bytes()) -> CollectionConverters.mapAsJavaMap(
```

Comment:
I think it is more efficient casting `proposition` to Addressproposition and calling directly `address()`

Reply:
outdated because the method was removed by a code refactor